lib/findMatchingFiles.js

```diff
@@ -45,3 +33,35 @@ export default function findMatchingFiles(
   // Filter out empty lines before returning
   return out.split('\n').filter((file: string): boolean => !!file);
 }
+
+function findMatchingFilesWithNode(
+  lookupPath: string,
+  validFilesRegex: string
+): Array<string> {
+  return glob.sync(`${lookupPath}/**/*.js*`, {
+    ignore: './node_modules/**/*',
+  }).filter((filePath: string): bool =>
+    new RegExp(validFilesRegex, 'i').test(filePath));
+}
+
+/**
+ * Finds files from the local file system matching the variable name.
+ */
+export default function findMatchingFiles(
+  lookupPath: string,
+  variableName: string
+): Array<string> {
+  if (lookupPath === '') {
+    // If lookupPath is an empty string, the `find` command will not work
+    // as desired so we bail early.
+    throw new Error(`lookup path cannot be empty (${lookupPath})`);
+  }
+
+  const formattedVarName = formattedToRegex(variableName);
+  const validFilesRegex = `(/|^)${formattedVarName}(/index)?(/package)?\\.js.*`;
+
+  if (/^win/.test(process.platform)) {
```

Comment:
If we don't see any meaningful performance differences between these methods, we might want to simplify and just use node for this.